docs/best-practices/continuous-integration/bitrise.md

```diff
@@ -0,0 +1,60 @@
+# Bitrise Integration
+
+[Bitrise](https://www.bitrise.io) allows you to automatically build, test and deploy your iOS and Android apps.
+
+## How to get started?
+
+Using _fastlane_ for your Bitrise workflow is simple. Just [add the ](https://devcenter.bitrise.io/getting-started/manage-your-bitrise-workflow) [_fastlane_](https://www.bitrise.io/integrations/steps/fastlane) [step to your
+workflow](https://devcenter.bitrise.io/getting-started/manage-your-bitrise-workflow),
+after the `Git Clone` step (and any other dependency step).
+
+### _Have our Certificate and profile installer step in your workflow!_
```

Comment:
Can we get rid of the italic here? It makes the whole paragraph a lot harder to read.

Reply:
Yup, fixed